### PR TITLE
Serverspec->Inspec

### PR DESCRIPTION
**Pull request by Vancelot11**

### CODE REVIEW

**Comment by Vancelot11** [comment]
[Cookbooks not yet migrated to Inspec](https://trello.com/c/HR86dvVI/1386-cookbooks-not-yet-migrated-to-inspec)

**Comment by Vancelot11** [comment]
@ramereth I removed the packages that were creating conflicts from the recipes but there aren't any tests in the current integration to confirm that no functionality was lost. If you have time would you mind checking out this PR and maybe recommending a way to test that it's still behaving as expected without the dkms package or cuda drivers?

**Comment by ramereth** [comment]
@Vancelot11 I fixed the issue in #37 so I recommend you rebase against that

**Comment by ramereth** [comment]
@Vancelot11 can you squash the last two commits into one please?

File: recipes/nvidia.rb [hunk 1]
```diff
@@ -93,4 +93,5 @@
   action :delete
 end
 
-package %w(nvidia-driver-latest-dkms cuda-drivers nvidia-docker2)
+package 'nvidia-docker2'
+# package %w(nvidia-driver-latest-dkms cuda-drivers nvidia-docker2)
```
Comment (ramereth) [inline recipes/nvidia.rb]: I fixed this in #38 so if you rebase, this should be reverted back.

File: test/integration/nvidia/inspec/nvidia_spec.rb [hunk 2]
```diff
@@ -4,20 +4,21 @@
...
 # found at http://inspec.io/docs/reference/resources/
-describe package('nvidia-driver-latest-dkms') do
-  it { should be_installed }
-  its('version') { should eq '418.87.01-1.el7' }
-end
+
+# describe package('nvidia-driver-latest-dkms') do
+#   it { should be_installed }
+#   its('version') { should eq '418.87.01-1.el7' }
+# end
```
Comment (ramereth) [inline test/integration/nvidia/inspec/nvidia_spec.rb]: Please add this back.

File: test/integration/nvidia/inspec/nvidia_spec.rb [hunk 3]
```diff
@@ -4,20 +4,21 @@
...
 end
 
-describe package('cuda-drivers') do
-  it { should be_installed }
-  its('version') { should eq '410.104-1' }
-end
+# describe package('cuda-drivers') do
+#   it { should be_installed }
+#   its('version') { should eq '410.104-1' }
+# end
```
Comment (ramereth) [inline test/integration/nvidia/inspec/nvidia_spec.rb]: Please add this back.

**Review by ramereth — CHANGES_REQUESTED** [review]

**Review by ramereth — APPROVED** [review]
Please merge as-is.

**Pull request merged**